workshop/README.md

```diff
@@ -0,0 +1,37 @@
+# An introduction to Kubernetes Service Catalog
+
+[Service Catalog](https://github.com/kubernetes-incubator/service-catalog) lets you provision cloud services and attach them to running Kubernetes deployments 
+without having to manage those resources directly. This is done through the use of a broker, a web
+server that implements the [Open Service Broker API](https://www.openservicebrokerapi.org/).
+
+# Objectives
+These labs are a introduction to the installation and use of Service Catalog inside a Kubernetes cluster. By the end of this course, you'll:
+* Understand the core concepts of the Open Service Broker API
+* Deploy an application with in memory state
+* Manually deploy a backing store
+* Manually attach the backing store to your application
+* Deploy a service broker, attach it to your Kubernetes deployment, and use it to automatically provision a backing store
+* Bind this OSB-provisioned backing store to your application running on Kubernetes
+
+# Prerequesites
+* Have completed [Kube101](https://github.com/IBM/kube101/tree/master/workshop).
+* Have a running Kubernetes cluster as per Kube101
+* A free-tier [IBM Cloud account](https://console.bluemix.net/registration/)
+
+# Open Service Broker API
+Cloud native applications are ephemeral and often have no local stable storage, but still require access to databases, caches,
+email systems, billing systems, etc. in order to function. OSBAPI is an open API that allows the concerns
```

Comment:
make it "Open Service Broker API (OSPAPI)" since you didn't introduce the acronym yet. unless I missed it before.